cime_config/usermods_dirs/PLUMBER2/defaults/user_nl_datm_streams

```diff
@@ -0,0 +1,51 @@
+!------------------------------------------------------------------------
+! This file is used to modify datm.streams.xml generated in $RUNDIR
+! Entries should have the form
+!  <streamname>:<stream_variable><= new stream_value>
+! The following are accepted values for an assume streamname of foo
+!  foo:meshfile = character string
+!  foo:datafiles = comma separated string of full pathnames (e.g. file1,file2,file3...)
+!  foo:datavars = comma separated string of field pairs  (e.g. foo foobar,foo2 foobar2...)
+!  foo:taxmode = one of [cycle, extend, limit]
+!  foo:tintalgo = one of [lower,upper,nearest,linear,coszen]
+!  foo:readmode = single (only suported mode right now)
+!  foo:mapalgo = one of [bilinear,redist,nn,consf,consd,none]
+!  foo:dtlimit = real (1.5 is default)
+!  foo:year_first = integer
+!  foo:year_last = integer
+!  foo:year_align = integer
+!  foo:vectors = one of [none,u:v]
+!  foo:lev_dimname: = one of [null,name of level dimenion name]
+!  foo:offset = integer
+! As an example:
+!   foo:year_first = 1950
+! would change the stream year_first stream_entry to 1950 for the foo stream block
+!------------------------------------------------------------------------
+CLM_USRDAT.CLM_USRDAT:datavars = ZBOT     z, \
+        TBOT     tbot, \
+        QBOT     shum, \
+        WIND     wind, \ 
+        PRECTmms precn, \ 
+        FSDS     swdn, \ 
+        PSRF     pbot, \ 
+        FLDS     lwdn
+
+! TODO, get right first and last years.
+
+presaero.SSP3-7.0:datafiles = $DIN_LOC_ROOT/atm/cam/chem/trop_mozart_aero/aero/aerodep_clm_SSP370_b.e21.BWSSP370cmip6.f09_g17.CMIP6-SSP3-7.0-WACCM.001_2018-2030_monthly_0.9x1.25_c210826.nc
+presaero.SSP3-7.0:year_first=2018
```

Comment:
@wwieder and @olyson I think this is something that would be good to discuss. If the solution we decide on is easy we could do it now, but otherwise we should make it into an issue and solve later.